.github/workflows/ci.yml

```diff
@@ -117,16 +119,21 @@ jobs:
       run: |
         cp .github/test_files/config_rasa_demo.yml rasa-demo/config.yml
 
+    - name: Set Rasa Version
+      run: |
+        RASA_VERSION=$(sed -ne 's/^rasa==\(.*\)#.*$/\1/p' rasa-demo/requirements.txt)
```

Comment:
`rasa-demo/requirements.txt`?

Does the file exist?

Reply:
We always run the test on the latest version of rasa-demo so I figured getting the rasa version directly from the repo could make our life easier. 

The rasa-demo repo has it here: https://github.com/RasaHQ/rasa-demo/blob/main/requirements.txt

Reply:
Ahh, I see. I didn't notice that we checkout the repo a few steps earlier.